packages/components/src/List/DisplayTable/DisplayTable.scss

```diff
@@ -2,6 +2,20 @@
 $tc-list-table-hover-bg: $concrete !default;
 $tc-list-table-color: $dove-gray !default;
 $tc-list-table-hover-color: $black !default;
+$tc-list-table-header-color: $black !default;
+$tc-list-table-header-height: 36px !default;
+
+.container {
+	padding-top: $tc-list-table-header-height;
+	position: relative;
+	display: flex;
+
+	> div {
+		width: 100%;
```

Comment:
Since its container has `display: flex` by default this `div`, as child, will be `flex: 1` so `width: 100%` is useless I guess.